tests/wpt/tests/css/css-text/white-space/pre-001.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<title>CSS-white-space test: restore collapsed whitespace</title>
+<link rel="author" title="JoeDow" href="ibluegalaxy_taoj@163.com">
+<link rel="match" href="reference/pre-001-ref.html">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-property">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+<style>
+html {
+  font: 20px/1 Ahem;
+}
+.span {
+    white-space: normal;
```

Comment:
`white-space: normal` is the default, can be removed.
Ditto below and in the ref file.